test/skimNtuple_HHbtag.cpp

```diff
@@ -3449,6 +3477,8 @@ int main (int argc, char** argv)
 
 			  else //eta region covered only by single lepton trigger
```

Comment:
I suppose this should mention the cross-triggers as well

Reply:
No, it is actually correct as it is. The ```else``` comes after the following ```else if```:

```c++
// eta region covered both by cross-trigger and single lepton trigger
else if(PERIOD != "2016preVFP" and PERIOD != "2016postVFP" and trgRegions["legacy"]) { ... }
```

and is supposed to be used only for 2016, where no cross-electron trigger is present. Note that we are inside the condition for the etau channel.